### PR TITLE
solution

### PR DESCRIPTION
**Pull request by MaxSchmide**
[DEMO LINK](https://maxschmide.github.io/react_carousel/)

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -20,17 +26,92 @@ class App extends React.Component<{}, State> {
...
+        <h1 data-cy='title' className='App__title'>Carousel with {images.length} images</h1>
+        <form className="App__form">
+          <div className="App__field">
+            <label htmlFor="animationDuration">Animation duration (ms)</label>
+            <input
+              type="number"
+              name="animationDuration"
+              id="animationDuration"
+              value={animationDuration}
+              onChange={(e) => this.handleInputChange(e)}
```
Comment (ilona-borkovska) [inline src/App.tsx:70]: This remark is connected with all your inputs: you should set limits for input values (-1 or 10000 are not valid nums for carousel)

**Review by ilona-borkovska — CHANGES_REQUESTED** [review]
Good job! But I am not sure that we should do this task by using class components

**Review by And678 — CHANGES_REQUESTED** [review]
Looks very nice, but tests need to pass https://github.com/mate-academy/react_carousel/pull/423/checks

**Review by And678 — APPROVED** [review]
Approve, discussed in slack

**Pull request closed without merging**